src/components/Counter2.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+//import '../css/App.css';
+
+class CounterTwo extends Component {
```

Comment:
Where is this being used?

Reply:
This component is not being used. This was me working with a class component instead of a function component, trying to get the javascript to run.